sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/ctl/AsyncCtlWorkload.java

```diff
@@ -0,0 +1,350 @@
+                logger.info("Deleted temporary collection {} created for this test", id);
+            }
+        }
+        cosmosClient.close();
+    }
+
+    private void performWorkload(BaseSubscriber<Object> documentSubscriber, OperationType type, long i) throws Exception {
+        Flux<? extends Object> obs;
+        CosmosAsyncContainer container = containers.get((int) i % containers.size());
+        if (type.equals(OperationType.Create)) {
+            PojoizedJson data = BenchmarkHelper.generateDocument(prefixUuidForCreate + i, dataFieldValue, partitionKey,
+                configuration.getDocumentDataFieldCount());
+            obs = container.createItem(data).flux();
+        } else if (type.equals(OperationType.Query)) {
+            CosmosQueryRequestOptions options = new CosmosQueryRequestOptions();
+            String sqlQuery = "Select top 100 * from c order by c._ts";
+            obs = container.queryItems(sqlQuery, options, PojoizedJson.class).byPage(10);
+        } else {
+            int index = random.nextInt(1000);
+            RequestOptions options = new RequestOptions();
+            String partitionKeyValue = docsToRead.get(container.getId()).get(index).getId();
+            options.setPartitionKey(new PartitionKey(partitionKeyValue));
+            obs = container.readItem(docsToRead.get(container.getId()).get(index).getId(),
+                new PartitionKey(partitionKeyValue),
+                PojoizedJson.class)
+                .flux();
+        }
+
+        concurrencyControlSemaphore.acquire();
+
+        obs.subscribeOn(Schedulers.parallel()).subscribe(documentSubscriber);
+    }
+
+    public void run() throws Exception {
+        readSuccessMeter = metricsRegistry.meter("#Read Successful Operations");
+        readFailureMeter = metricsRegistry.meter("#Read Unsuccessful Operations");
+        writeSuccessMeter = metricsRegistry.meter("#Write Successful Operations");
+        writeFailureMeter = metricsRegistry.meter("#Write Unsuccessful Operations");
+        querySuccessMeter = metricsRegistry.meter("#Query Successful Operations");
+        queryFailureMeter = metricsRegistry.meter("#Query Unsuccessful Operations");
+        readLatency = metricsRegistry.timer("Read Latency");
+        writeLatency = metricsRegistry.timer("Write Latency");
+        queryLatency = metricsRegistry.timer("Query Latency");
+
+        reporter.start(configuration.getPrintingInterval(), TimeUnit.SECONDS);
+        long startTime = System.currentTimeMillis();
+
+        AtomicLong count = new AtomicLong(0);
+        long i;
+        int writeRange = readPct + writePct;
+        for (i = 0; BenchmarkHelper.shouldContinue(startTime, i, configuration); i++) {
+            int index = (int) i % 100;
+            if (index < readPct) {
+                BenchmarkRequestSubscriber<Object> readSubscriber = new BenchmarkRequestSubscriber<>(readSuccessMeter,
+                    readFailureMeter, concurrencyControlSemaphore, count);
```

Comment:
as a code style, we should try to either 
1. have all args on the same line
2. or if there are many, have one arg per line. 

Please try to follow that here and in other new code.

Reply:
done